Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '2.6.3'
+ruby '>=2.6.3'
```

Comment:
**Inconsistency ruby version:** your ruby version in your `Gemfile` and your `.ruby-version `file should be the same to avoid  crashing errors

Reply:
Changes are done! Thank you for your review!